client/landing/gutenboarding/hooks/use-plans-billing-period.ts

```diff
@@ -0,0 +1,3 @@
+export function usePlansBillingPeriod(): 'ANNUALLY' | 'MONTHLY' {
```

Comment:
Isn't there a way to avoid bringing the billing period data into Gutenboarding (and launch flows) and keep it only at the level of PlansGrid ?

All annual plans have monthly plans as a direct equivalent so I expect we can continue to keep the consumer application (and even the logic to get recommended / popular plan) unaware that monthly plans even exist. 

The only difference should be when a user picks a monthly plan from Plans Grid => the returned plan should have the correct `productId`.

Reply:
There is also the price difference and the refund period.

Reply:
Both are displayed only inside PlansGrid so knowing the billing period at that level to get correct Plan object should be enough.